### PR TITLE
Adding intial aerosol regression test of CATChem

### PR DESCRIPTION
**Pull request by bbakernoaa**
## Description
This PR adds an initial regression test allowing for rapid updates and ensuring that the model is stable and any changes can be traced down easily. This is related to https://github.com/ufs-community/CATChem/issues/7

### Input data additions/changes
- [x] No changes are expected to input data.
- [x] There will be new input data. 
- [x] Input data will be updated. 

### Anticipated changes to regression tests:
- [x] Changes are expected to the following tests: 

New regression test 

## Subcomponents involved:
- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [x] FV3
- [ ] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [x] catchem
- [ ] none

### CODE REVIEW

**Comment by bbakernoaa** [comment]
note that there is an issue with dust5 tracer.  It happens with both the AFWA and FENGSHA dust code so it must be somewhere in the driver. But I suggest we go ahead and proceed as this is an initial testing site

## To use for those that don't know 

```bash 
cd ufs-weather-model/tests

./rt.sh -k -n atmcataero_control_p8
```

Note you may have to update your account in `rt.sh`

File: tests/fv3_conf/control_run.IN [hunk 1]
```diff
@@ -143,3 +143,10 @@ if [ $CPLCHM = .true. ]; then
   cp      ${PATHRT}/parm/gocart/*.rc .
   ln -sf  @[INPUTDATA_ROOT]/GOCART/p8/ExtData .
 fi
+
+
+
+#CATChem Aerosol Inputs 
+#if [ $CPLCHP = .true. ]; then # <<<< This needs fixed. should be an option set in the tests/atmcataero_control_p8 file. for some reason it breaks
```
Comment (bbakernoaa) [inline tests/fv3_conf/control_run.IN:150]: Not sure why this failed but we need to figure out how to fix it later

**Comment by jianheACM** [comment]
@bbakernoaa Thanks for doing this. Will look into more details later.

**Comment by zmoon** [comment]
@jianheACM the diff showed that these files in this PR were missing an ending newline. Should be added.

- tests/fv3_conf/control_run.IN
- tests/parm/field_table/field_table_thompson_noaero_tke_cataero
- tests/rt.conf

**Comment by jianheACM** [comment]
@zmoon Thanks Zack! I have fixed ending newlines in the above three files.

**Pull request merged**